src/components/sidenav/sidenav.scss

```diff
@@ -0,0 +1,141 @@
+@import "default-theme";
+@import "mixins";
+@import "variables";
+@import "shadows";
+
+
+$md-sidenav-backdrop-color: invert(md-color($md-background, card, 0.6)) !default;
+$md-sidenav-background-color: md-color($md-background, dialog) !default;
+$md-sidenav-push-background-color: md-color($md-background, dialog) !default;
+
+
+/**
+ * Mixin to help with defining LTR/RTL `transform: translateX()` values.
+ * @param $open The translation value when the sidenav is opened.
+ * @param $close The translation value when the sidenav is closed.
+ */
+@mixin md-sidenav-transition($open, $close) {
+  transform: translateX($close);
+
+  &.md-sidenav-closed {
+    // We use `visibility: hidden | visible` because `display: none` will not animate any
+    // transitions, while visibility will interpolate transitions properly.
+    // see https://developer.mozilla.org/en-US/docs/Web/CSS/visibility, the Interpolation
+    // section.
+    visibility: hidden;
```

Comment:
Will this cause any issues with the invisible element unintentionally catching user clicks?